src/Tribe/Log/Service_Provider.php

```diff
@@ -145,4 +147,30 @@ public function add_logging_engine( array $logging_engines = [] ) {
 
 		return $logging_engines;
 	}
+
+	/**
+	 * Enables logging in the service provider, if not already enabled.
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 8b31288728b23558c1aba48ff410604f</sub>